php/ice/config.yaml

```diff
@@ -0,0 +1,23 @@
+framework:
+  website: iceframework.org
+  version: 1.5.1
+
+build_deps:
+  - bash
+
+before_build:
+  - wget -O /tmp/framework-1.5.1.tar.gz https://github.com/ice/framework/archive/1.5.1.tar.gz
+  - cd /tmp/ && tar -xzf framework-1.5.1.tar.gz && cd framework-1.5.1 && bash install
+  - cp /tmp/ framework-1.5.1/tests/ci/ice.ini /usr/local/etc/php/conf.d/docker-php-ext-ice.ini
```

Comment:
is this config file tuned for CI ? could it be tuned for a real-word app ?

Reply:
It's just to turn the extension on:
```ini
extension=ice.so
```

Reply:
Rather use an `echo extension=ice.so` then. 

I find it more explicit